app/models/solidus_stripe/payment_intent.rb

```diff
@@ -89,7 +98,7 @@ def reload(...)
       super
     end
 
-    def create_stripe_intent(**stripe_intent_options)
+    def create_stripe_intent(payment, **stripe_intent_options)
```

Comment:
nit: should this be a private method or maybe inlined?

Reply:
Moving this method to private involves some changes 🤔 
It's called in the instance created here:
https://github.com/solidusio/solidus_stripe/blob/bb262eb8e282f77cc6d1ca55f6c3e2a37e6500a3/app/models/solidus_stripe/payment_intent.rb#L20
And some options need some other adjustments because they come from instance methods and associations.

I'd rather leave it public, but let me know what you think 👍

Reply:
That's ok then 👍